.devcontainer/base-dev/base-dev.Dockerfile

```diff
@@ -377,10 +369,13 @@ ENV DEBIAN_FRONTEND=
 
 # install dev python3 dependencies
 RUN pip3 install \
-    # to be able to run juypter notebooks
+    # for juypter notebooks
     ipykernel \
     # for integration_tests package
-    types-PyYAML
+    types-PyYAML \
+    # for local pathfinding
+    plotly \
+    flask
```

Comment:
Should pip install these packages in the notebook itself, for example: https://github.com/UBCSailbot/notebooks/blob/main/local_pathfinding/coord_systems.ipynb

Reply:
Which notebook do you mean?

Reply:
nvm these are the development-only dependencies of local pathfinding. See my commit in your local pathfinding pr